torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from torch.fx.graph import map_arg
+
+
+def get_all_users_of(index: int, fx_module: GraphModule) -> List[int]:
+    "Given the graph(fx_module) and an index, return a list of all node indexes that use this node"
+    graph = fx_module.graph
+    node = graph.nodes[index]
+    user_indexes: List[int] = []
+    """if the node A is in node B's args, then B is the user of A
+       go through all the nodes, if the input node in any node's args,
+       then that node is the input node's user
+    """
+    for index, n in enumerate(graph.nodes):
+        args = map_arg(n.args, lambda n: n)
+        kwargs = map_arg(node.kwargs, lambda n: n)
```

Comment:
These lines aren't actually doing anything, the purpose of map_arg is to transform `Node` values within nested data structures.

What actually needs to happen here is you need to recursively traverse the `args` and `kwargs` data structures and return `True` if the node of interest is contained anywhere in there. Something like:
```
        def find_use(arg : Any) -> bool:
            if isinstance(arg, (tuple, list)):
                return any(find_use(elem) for elem in arg)
            elif isinstance(arg, dict):
                return any(find_use(v) for k, v in arg.items())
            elif isinstance(arg, slice):
                return any(find_use(arg.start), find_use(arg.stop), find_use(arg.step))
            elif isinstance(arg, Node):
                return arg is node
            else:
                return False

        return [n for n in self._nodes if find_use(n.args) or find_use(n.kwargs)]
```

Note I actually didn't use `map_arg` here, since we're trying to reduce the structure down rather than map it through.